eskip/string_test.go

```diff
@@ -100,6 +100,16 @@ func TestRouteString(t *testing.T) {
 			Filters:     []*Filter{{"filter0", []interface{}{"arg"}}},
 			BackendType: DynamicBackend},
 		`* -> filter0("arg") -> <dynamic>`,
+	}, {
+		&Route{
+			Filters:     []*Filter{{"filter0", []interface{}{`Line 1\r\nLine 2`}}},
```

Comment:
This is wrong due to https://github.com/zalando/skipper/pull/1341/files#r773092391, the raw `Line 1\r\nLine 2` should be escaped as `"Line 1\\r\\nLine 2"`